lib/doorkeeper/helpers/controller.rb

```diff
@@ -45,6 +45,8 @@ def get_error_response_from_exception(exception)
                        :invalid_request
                      when Errors::InvalidGrantReuse
                        :invalid_grant
+                     else
+                       exception.message
```

Comment:
Agree. You mean something like this right?

```ruby
...
when Errors::DoorkeeperError
  exception.message
end
```
So if a different error is raised we bubble it up.